lib/constants/videoState.js

```diff
@@ -1,17 +1,19 @@
+/* eslint-disable */
+
 const videoState = {
-	STOPPED: Symbol('stopped'),
-	ERROR: Symbol('error'),
-	PLAYING: Symbol('playing'),
-	PAUSED: Symbol('paused'),
-	CONNECTING: Symbol('connecting'),
-	BUFFERING: Symbol('buffering'),
-	FINISHED: Symbol('finished'),
-	IDLE: Symbol('idle'),
-	PREPARING: Symbol('preparing'),
-	PREPARED: Symbol('prepared'),
-	PLAYBACK_COMPLETED: Symbol('playback_completed'),
-	UNKNOWN: Symbol('unknown'),
-	REVERSING: Symbol('reversing'),
+	STOPPED:            'stopped',
+	ERROR:              'error',
+	PLAYING:            'playing',
+	PAUSED:             'paused',
+	CONNECTING:         'connecting',
+	BUFFERING:          'buffering',
+	FINISHED:           'finished',
+	IDLE:               'idle',
+	PREPARING:          'preparing',
+	PREPARED:           'prepared',
+	PLAYBACK_COMPLETED: 'playback_completed',
```

Comment:
Please, use camelCase everywhere.

Reply:
Do you mean change properties names like `playbackCompleted: 'playback_completed'` and  `prepared: 'prepared'`?, it will brake existing tests

Reply:
It should be:
PLAYBACK_COMPLETED: 'playbackCompleted'

Reply:
Should vrc buttons and network methods be in camelCase as well?

Reply:
Nope, those are all uppercase. all constants should be either camelCase or UPPER_CASE_WITH_DASH, but never lower_case_with_dash